pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = /Users/eslothower/opt/anaconda3/bin
```

Comment:
This file seems to have been added accidentally (especially with the username in the filepath...)